src/init.cpp

```diff
@@ -1368,10 +1360,18 @@ bool AppInitMain(const util::Ref& context, NodeContext& node, interfaces::BlockA
     node.banman = MakeUnique<BanMan>(GetDataDir() / "banlist.dat", &uiInterface, args.GetArg("-bantime", DEFAULT_MISBEHAVING_BANTIME));
     assert(!node.connman);
     node.connman = MakeUnique<CConnman>(GetRand(std::numeric_limits<uint64_t>::max()), GetRand(std::numeric_limits<uint64_t>::max()), args.GetBoolArg("-networkactive", true));
+
     // Make mempool generally available in the node context. For example the connection manager, wallet, or RPC threads,
```

Comment:
This comment is outdated (the mempool is being constructed, not just made available). If you retouch the PR, can you update the comment?

Reply:
Good point. The comment should probably be removed because now that it is no longer a global, it is clear from the code where it is passed in.